### DISCUSSION TITLE
Sorry the cheat sheet URL is unable to access. Is the server down?

### DISCUSSION THREAD

**Discussion by kabir-ibi** [thedevdojo/voyager#5818] [Q&A]
I tried to access this URL but unable to access.
View the Voyager Cheat Sheet: https://voyager-cheatsheet.ulties.com/

**Comment by mavisland** [comment]
> I tried to access this URL but unable to access. View the Voyager Cheat Sheet: https://voyager-cheatsheet.ulties.com/

The site seems to have been suspended and closed a long time ago. Maybe this backup will work for you.

https://web.archive.org/web/20220523182253/https://voyager-cheatsheet.ulties.com/

**Reply by kabir-ibi** [reply]
@mavisland  Thank you so much for the backup. It will help me a lot.
